src/requirements/data/colleges/en.ts

```diff
@@ -5,6 +5,23 @@ import {
   includesWithSubRequirements,
 } from '../checkers-common';
 
+const engineeringLiberalArtsDistributions: readonly string[] = [
+  'CA',
+  'HA',
+  'LA',
+  'LAD',
+  'KCM',
+  'SBA',
+  'FL',
+  'CE',
+  'ALC',
+  'SCD',
+  'HST',
+  'ETM',
+  'SSC',
+  'GLC',
```

Comment:
Added these new categories. It turns out that 2019 and 2020 requirements for engineering is exactly the same...
https://www.engineering.cornell.edu/students/undergraduate-students/advising/liberal-studies/liberal-studies-policy-students-entering
https://www.engineering.cornell.edu/students/undergraduate-students/advising/liberal-studies/liberal-studies-policy-students-entering-0
not sure why they made two separate pages...